### PR TITLE
JZ script for experiments on Entity

### PR DESCRIPTION
**Pull request by manandey**

### CODE REVIEW

File: experiments/jz/entity/exp_1/toy_example/03_create_dataset.slurm [hunk 1]
```diff
@@ -47,10 +47,9 @@ REPO_DIR="${DATASETS_CUSTOM}/${myarray[-1]}"
 python bsmetadata/train.py \
   data_config.experiment="with_metadata" \
   data_config.metadata_config.metadata_list=["entity"] \
-  data_config.metadata_config.max_seq_len=1024 \
```
Comment (SaulLu) [inline experiments/jz/entity/exp_1/toy_example/03_create_dataset.slurm]: Out of curiosity, why did you remove this line? :blush:

File: experiments/jz/entity/exp_1/toy_example/02_load_dataset.slurm [hunk 2]
```diff
@@ -56,5 +56,5 @@ cd $WORK/repos/sync/metadata/
 # We check that the dataset can indeed be loaded
 python experiments/jz/utils/loading_script_utils/load_dataset.py \
     dataset_name="${REPO_DIR}" \
-    train_file="oscar_train.jsonl" \
-    validation_file="oscar_dev.jsonl"
+    train_file="oscar_train.jsonl.gz" \
+    validation_file="oscar_dev.jsonl.gz"
```
Comment (SaulLu) [inline experiments/jz/entity/exp_1/toy_example/02_load_dataset.slurm:60]: Thank you :hugs:

File: experiments/jz/entity/exp_1/toy_example/03_create_dataset.slurm (hunk 1 above)
Comment (SaulLu) [reply experiments/jz/entity/exp_1/toy_example/03_create_dataset.slurm]: It's really as you wish for your experiences (I don't have a preference)! It's just that the gpt-2 model supports up to 1024 and wanted to make sure it wasn't from "bad" information that was given by the default settings.

Comment (manandey) [reply experiments/jz/entity/exp_1/toy_example/03_create_dataset.slurm]: I didn't face any such issues as of now. Since, gpt-2 supports upto 1024, I'd prefer to use the same as well.

**Review by SaulLu — APPROVED** [review]
Manan, thank you very much for this PR! Everything looks very good and ready to be tested. :rocket:
